coordinator/src/services/translation/request/send-message-payload.ts

```diff
@@ -73,14 +73,9 @@ function convertRequesterToControlActAuthor(
   bundle: fhir.Bundle
 ) {
 
-  // todo dispenseNotification: implement dispense verson
   const messageType = identifyMessageType(bundle)
   if (messageType === fhir.EventCodingCode.DISPENSE) {
```

Comment:
If dispense notifications should use the unattended author details, should the translations just call `createControlActEventForUnattendedAccess()` instead of `createControlActEvent()`? Then we can remove this bespoke logic.